index.js

```diff
@@ -13,9 +13,9 @@ try {
  * Defer.
  */
 
-var defer = typeof process === 'undefined' || typeof process !== 'function'
-  ? function(fn){ setTimeout(fn); }
-  : process.nextTick;
+var defer = typeof process !== 'undefined' && typeof process.nextTick === 'function'
```

Comment:
the language evaluates left-to-right, so the second statement won't ever be evaluated if `process === undefined`.

if it's `null`, you're right, there's an issue here, but i can't think of any practical case where `process === null`.